concepts/05 UI Components/TextArea/00 Getting Started with TextArea/15 Resize TextArea.md

```diff
@@ -0,0 +1,59 @@
+If the size of the UI component should be fixed, specify the [height](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#height) and [width](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#width) properties.
```

Comment:
A minor note... but it's more natural to put "width" first. X and Y, width and height - they usually go in that order. 

```suggestion
If the size of the UI component should be fixed, specify the [width](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#width) and [height](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#height) properties.
```